conduit/plugins/importers/algod/algod_importer.go

```diff
@@ -440,10 +456,7 @@ func waitForRoundWithTimeout(ctx context.Context, l *logrus.Logger, c *algod.Cli
 		if rnd <= status.LastRound {
 			return status.LastRound, nil
 		}
-		return 0, &SyncError{
-			rnd:      status.LastRound,
-			expected: rnd,
-		}
+		return 0, NewSyncError(status.LastRound, rnd, fmt.Errorf("this check should never be required: %w", err))
```

Comment:
```suggestion
		// algod's timeout should not be reached because context.WithTimeout is used
		return 0, NewSyncError(status.LastRound, rnd, fmt.Errorf("sync error, likely due to status after block timeout: %w", err))
```